src/test/java/org/reaktivity/nukleus/maven/plugin/internal/generated/ListFW.java

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright 2016-2019 The Reaktivity Project
+ *
+ * The Reaktivity Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package org.reaktivity.nukleus.maven.plugin.internal.generated;
+
+import org.agrona.DirectBuffer;
+import org.agrona.MutableDirectBuffer;
+import org.reaktivity.reaktor.internal.test.types.Flyweight;
+
+public abstract class ListFW extends Flyweight
+{
+    public abstract int length();
+
+    public abstract int fieldCount();
+
+    public abstract DirectBuffer fields();
+
+    public abstract static class Builder<T extends ListFW> extends Flyweight.Builder<T>
+    {
+        private int fieldCount;
+
+        public Builder(T flyweight)
+        {
+            super(flyweight);
+        }
+
+        public Builder<T> set(
+            ListFW value)
+        {
+            fieldCount += value.fieldCount();
+            return this;
+        }
```

Comment:
I originally had this `set` method as an abstract method and both `List32FW.Builder` and `List8FW.Builder` overrides the `set` method.
Do you mean we should remove this `set` (abstract) from `ListFW` completely and let subclasses write their own `set`?

Reply:
I misread this and the other feedback at first but after reading all the feedback, I think I understand it now. So we basically use `fields()` instead of `set()` wherever `set()` is used. So we don't need this `set` in all the `List#FW` too.